circus-train-integration-tests/pom.xml

```diff
@@ -188,6 +206,39 @@
       <version>1.7.7</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.avro</groupId>
+      <artifactId>avro-mapred</artifactId>
+      <version>1.7.7</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
+      <version>3.1.0</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.esotericsoftware</groupId>
+      <artifactId>kryo</artifactId>
+      <version>4.0.2</version>
```

Comment:
Should this be test or provided scope? I'd be concerned of possible classpath clash if this ends up in CT where it didn't beore.

Reply:
will move this to the test scope